src/forms/renderers/CollapsibleGroup.tsx

```diff
@@ -0,0 +1,51 @@
+import { RankedTester, rankWith, uiTypeIs } from '@jsonforms/core';
+import {
+    MaterialLayoutRenderer,
+    MaterialLayoutRendererProps,
+} from '@jsonforms/material-renderers';
+import { withJsonFormsLayoutProps } from '@jsonforms/react';
+import ExpandMoreIcon from '@mui/icons-material/ExpandMore';
+import {
+    Accordion,
+    AccordionDetails,
+    AccordionSummary,
+    Hidden,
+    Typography,
+} from '@mui/material';
+
+export const collapsibleGroupTester: RankedTester = rankWith(
+    999,
+    uiTypeIs('CollapsibleGroup')
+);
+
+const CollapsibleGroupRenderer = (props: any) => {
+    const { uischema, schema, path, visible, renderers } = props;
+
+    const labelElement = uischema.elements.shift();
```

Comment:
Since I know we add a label to this type I am pulling it out here. Otherwise the collapsible portion would end up having a double label like this:

![image](https://user-images.githubusercontent.com/270078/153654353-f52de9a2-82fa-429a-ada4-25cc2809ef49.png)